charts/k8s-monitoring/templates/agent_config/_grafana_operator_objects.txt

```diff
@@ -0,0 +1,79 @@
+{{ define "agent.config.pod_log_objects" }}
+// PodMonitor
+loki.source.podlogs "pod_logs" {
```

Comment:
This isn't referenced again in this file, is it still needed?

Reply:
Oop, yes. Thanks. Removed that.